### PR TITLE
some tests

### PR DESCRIPTION
**Pull request by shcherbyakdev**

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+
+describe('Validation inputs',()=> {
+    describe('name',()=>{
+        test('is type of name-field equals string', ()=> {
+            expect(validations.validateName('museum')).toBe('museum');
+         })
+    })
+    describe('radius',()=>{
+        test('is radius comvert to meters', ()=>{
```
Comment (vladholubiev) [inline test/test.js]: typo `comvert`

File: package.json [hunk 2]
```diff
@@ -20,5 +20,8 @@
     "json2csv": "^4.0.2",
     "request": "^2.85.0",
     "validator": "^9.4.1"
+  },
+  "devDependencies": {
+    "jest": "^22.4.3"
```
Comment (KnupMan) [inline package.json]: `package-lock` file should be updated as well

File: test/test.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            expect(validations.validateRadius(100)).toBe(100000);
+        })
+        test('is radius number', () =>{
+            expect(validations.validateRadius('100')).not.toBe(10000);
```
Comment (vladholubiev) [inline test/test.js]: Shouldn't it throw an error if we pass a string?

File: test/test.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+    describe('Get inputs from user', ()=> {
+        test('if function return validate object', ()=> {
+            expect(helpersFunctions.processInputsData({"query": "museum",
+            "radius": 100,
+            "latitude": 40.7,
+            "longitude": -74.01})).toEqual({"venueType": "museum",
+            "radius": 100000,
+            "latitude": 40.700000,
+            "longitude": -74.010000})
```
Comment (vladgolubev) [inline test/test.js]: What do you expect? Looks like no `toBe` or `toEqual` in the end

**Review by vladholubiev — APPROVED** [review]

**Review by KnupMan — APPROVED** [review]

**Review by KnupMan — APPROVED** [review]

**Review by vladholubiev — APPROVED** [review]

**Pull request merged**